qa/src/test/java/tests/ApiTest.java

```diff
@@ -1,106 +1,45 @@
-        JSONObject jsonObjectID = jsonObjectData.getJSONObject("3843");
-        System.out.println(jsonObjectID.toString());
-        String symbol = jsonObjectID.getString("symbol");
-        System.out.println(symbol);
-        Assert.assertEquals(symbol,"BOLT");
-        String dateAdded = jsonObjectID.getString("date_added");
-        System.out.println(dateAdded);
-        Assert.assertEquals(dateAdded, "2019-04-05T00:00:00.000Z");
-        String logo = jsonObjectID.getString("logo");
-        System.out.println(logo);
-        Assert.assertEquals(logo, "https://s2.coinmarketcap.com/static/img/coins/64x64/3843.png");
-
-        JSONObject jsonObjectURLs = jsonObjectData.getJSONObject("3843").getJSONObject("urls");
-        System.out.println(jsonObjectURLs.toString());
-        JSONArray urlsArray = jsonObjectURLs.getJSONArray("source_code");
-        String source_code = urlsArray.get(0).toString();
-        System.out.println(source_code);
-        Assert.assertEquals("https://github.com/SyQic-Ops/bolt",source_code);
-
-        JSONArray websiteArray = jsonObjectURLs.getJSONArray("website");
-        String website = websiteArray.get(0).toString();
-        System.out.println(website);
-        Assert.assertEquals("https://bolt.global/",website);
-    }
-
     @Test
-    public void apiGET()
-    {
-        RestAssured.baseURI ="https://jsonplaceholder.typicode.com/";
-
-        RequestSpecification httpRequest = given();
-        Response response = httpRequest.request(Method.GET, "/users");
-
-        String responseBody = response.getBody().asString();
-        System.out.println("Response Body is =>  " + responseBody);
-
-        System.out.println("status code:"+response.getStatusCode());
-        Assert.assertEquals("Status code is not 200",200,response.getStatusCode());
-    }
-
-    @Test
-    public void apiPOST()
-    {
-        String baseuri = BaseHelperPropertieManager.getInstance().getURI("/users");
-        baseURI = baseuri;
-
-        JSONObject requestParams = new JSONObject();
-        requestParams.put("email", "srdjan.rados@htecgroup.com");
-        requestParams.put("password", "Qwertysha1@");
-        requestParams.put("firstName",  "Srdjan");
-        requestParams.put("lastName",  "Rados");
-
-        Response postResponse = given()
-                .contentType(ContentType.JSON)
-                .body(requestParams.toString()).log().all()
-                .post("/register");
-
-        System.out.println("----------------");
-        System.out.println("Post response is "+postResponse.asString());
-        System.out.println("----------------");
-        System.out.println("Post response status is "+postResponse.statusCode());
-
+    public void apiPostUserRegisteredCheck() throws IOException {
+
+        ExcelReader excelReader = new ExcelReader("src/main/resources/ExcelRead.xlsx");
+        
+        String email = excelReader.getStringData("apiTest", 1, 0);
```

Comment:
Ovo citanje iz excela i dodeljivanje vrednosti mozete uraditi u samoj User klasi, npr metodi set values for valid user. Ukoliko cete cesto to koristiti da ne pisete sto puta 4 linije email, pass itd